src/main/java/com/zaxxer/hikari/pool/PoolInitializationException.java

```diff
@@ -31,6 +31,6 @@
     */
    public PoolInitializationException(Throwable t)
    {
-      super("Exception during pool initialization", t);
+      super(t);
```

Comment:
That's going to weird in a log, especially if the user suppresses stack traces.  If you're going to change it, this would be better:
```java
super("Exception during pool initialization: " + t.getMessage(), t);
```